src/core/include/openvino/op/grid_sample.hpp

```diff
@@ -30,6 +30,9 @@ class OPENVINO_API GridSample : public Op {
         InterpolationMode mode = InterpolationMode::BILINEAR;
         // Specifies how the out-of-bounds coordinates should be handled: `zeros`, `border`, `reflection`
         PaddingMode padding_mode = PaddingMode::ZEROS;
+
+        Attributes() = default;
+        Attributes(InterpolationMode m, PaddingMode p, bool a) : align_corners{a}, mode{m}, padding_mode{p} {}
```

Comment:
Can you chance "m", "p" and "a" to full names matching the initialized members? Also I think the order o this constructor's arguments could reflect the order of members in the struct.

Reply:
Done.